components/raftstore/src/store/util.rs

```diff
@@ -1158,10 +1160,16 @@ fn check_remove_or_demote_voter(
         if change_type == ConfChangeType::RemoveNode
             || change_type == ConfChangeType::AddLearnerNode
         {
+            let is_voter = region
```

Comment:
Recommend renaming `normal_peers_to_remove` and `slow_peers_count` to `normal_voters_to_remove` and `slow_voters_count`

Reply:
#16476 16476